### PR TITLE
changed structure of admin-login page and worked on responsiveness

### PR DESCRIPTION
**Pull request by priyalpepcus**

### CODE REVIEW

File: src/card-print.css [hunk 1]
```diff
@@ -1,158 +1,54 @@
 @media print {
-    body,html, #root {
-        height:100%;
-        width:100%;
-        /*-webkit-print-color-adjust: exact;*/
+    @font-face {
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: Define font family in desktop based CSS file not in @media print

File: src/card-print.css [hunk 2]
```diff
@@ -1,158 +1,54 @@
...
+
+    body,html, #root {
+       /* height:100%;
+        width:100%;*/
+        -webkit-print-color-adjust: exact;
+        font-family: "Roboto Condensed", Serif;
     }
 
+    .grid-scroll-page-wrapper {
+        overflow-y: inherit;
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: Why we are using inherit here?

File: src/card-print.css [hunk 3]
```diff
@@ -1,158 +1,54 @@
...
+        min-height: 100%;
+        /* margin-top: 0px !important; */
+    }
+    .grid-scroll-wrapper {
+        overflow-y: inherit;
+        margin-bottom: 0;
+        display:block;
+        margin-top: 0;
+    }
     .student-grid-none {
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: change class name to print-media-none from stdent-grid-none

File: src/card-print.css [hunk 4]
```diff
@@ -1,158 +1,54 @@
...
         display:none;
     }
-    .student-id-cards{
-        border: 1px solid #d94803;
-        width:45%;
-        box-sizing:border-box;
-        margin:0px 15px 20px;
+    .id-card-wrapper {
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: Add media-print-none class where we need to hide for media print css.

File: src/card-print.css [hunk 5]
```diff
@@ -1,158 +1,54 @@
...
+        display:block;
+        margin-top: 0;
+    }
     .student-grid-none {
         display:none;
     }
-    .student-id-cards{
-        border: 1px solid #d94803;
-        width:45%;
-        box-sizing:border-box;
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: why we need to add box sizing in media print.

File: src/card-print.css [hunk 6]
```diff
@@ -1,158 +1,54 @@
...
-        margin:4px 0px 10px;
-    }
-    .card-fields-wrapper .card-text-spacing {
-        flex:0.5 0 0;
-        margin:4px 0px 10px;
-    }
-    .student-id-cards-header{
-        text-align: center;
-        font-size: 18px;
-        color: white !important;
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: use hexadecimal values

File: src/card-print.css (hunk 1 above)
Comment (priyalpepcus) [reply src/card-print.css]: Removed font-family from **@media** print CSS file

File: src/components/SplashPage.jsx [hunk 7]
```diff
@@ -290,8 +290,10 @@ class SplashPage extends Component {
         return <Switch><Redirect to={'/studentCorrection'} /></Switch>
       }
     return (
+      <div className="landing-page-full-height">
```
Comment (avi-jain-pepcus) [inline src/components/SplashPage.jsx]: change class name and give proper descriptive name

File: src/dataGrid.css [hunk 8]
```diff
@@ -71,6 +71,21 @@ html, body {
     width: 100%;
     text-align: center;
 }
+.landing-page-full-height {
+    /*height:auto;*/
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: remove unnecessary comments from CSS file.

File: src/card-print.css (hunk 2 above)
Comment (priyalpepcus) [reply src/card-print.css]: I have removed unnecessary overflow-y property in **@media** print CSS file as such no used in here.

File: src/dataGrid.css [hunk 9]
```diff
@@ -2078,11 +2241,12 @@ body,
...
-    .yjsg-event-info {
-        margin:5px 0 0;
+    .landing-page-wrapper {
+        display:block;
+        padding:40px 0;
     }
-}
 
+}
 @media (max-width: 425px) {
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: remove this media query and please use standard media queries like 480, 640, 768 & 1024.

File: src/dataGrid.css [hunk 10]
```diff
@@ -1534,14 +1546,159 @@ body,
...
+}
+.card-text-bus-stop {
+    height:10px;
+}
+.card-flex {
+    display: flex;
+    align-items: center;
+}
+
 @media (max-width: 1200px) {
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: remove this media query as well.

File: src/dataGrid.css [hunk 11]
```diff
@@ -1820,6 +1977,11 @@ body,
...
         min-width: 110px;
     }
+    .landing-page-full-height {
+        height:100%;
+        display:flex;
+        flex-direction: column;
+    }
 }
 
 @media (max-width: 600px) {
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: remove this media query as well.

File: src/card-print.css (hunk 3 above)
Comment (priyalpepcus) [reply src/card-print.css]: Replaced class name student-grid-none to print-media-none.

File: src/dataGrid.css [hunk 12]
```diff
@@ -2214,6 +2385,9 @@ body,
     .footer {
         padding: 10px 0;
     }
+    .landingPageContent {
+        margin-bottom:20px;
+    }
 }
 
 @media (max-width: 280px) {
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: remove this media query as well.

File: src/card-print.css (hunk 4 above)
Comment (priyalpepcus) [reply src/card-print.css]: I have added there.

File: src/card-print.css (hunk 5 above)
Comment (priyalpepcus) [reply src/card-print.css]: This code is removed in card-print CSS file.

File: src/card-print.css (hunk 6 above)
Comment (priyalpepcus) [reply src/card-print.css]: I have changed color white to hex code(#ffffff) in CSS file

File: src/components/SplashPage.jsx (hunk 7 above)
Comment (priyalpepcus) [reply src/components/SplashPage.jsx]: I have changed class name landing-page-full-height to landing-page-block.

File: src/dataGrid.css (hunk 8 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed unnecessary comments from CSS file.

File: src/dataGrid.css (hunk 9 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed media query 425 to 480 and 500 to 640.

File: src/dataGrid.css (hunk 10 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed.

File: src/dataGrid.css (hunk 11 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed.

File: src/dataGrid.css (hunk 12 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed.

File: src/card-print.css [hunk 13]
```diff
@@ -47,9 +28,6 @@
     {
```
Comment (avi-jain-pepcus) [inline src/card-print.css]: remove .noprint.

Comment (priyalpepcus) [reply src/card-print.css]: Removed.

**Pull request merged**